client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx

```diff
@@ -1,65 +1,41 @@
 import InvestigationInfoBar from './InvestigationInfo/InvestigationInfoBar';
-import { StartInvestigationDateVariablesProvider, StartInvestigationDateVariables } from './StartInvestigationDateVariables/StartInvestigationDateVariables';
 
 export const LAST_TAB_ID = 3;
 export const END_INVESTIGATION = 'סיים חקירה';
 export const CONTINUE_TO_NEXT_TAB = 'המשך לשלב הבא';
 
 const InvestigationForm: React.FC = (): JSX.Element => {
     const classes = useStyles({});
-    const [exposureDate, setExposureDate] = React.useState<Date>();
-    const [symptomsStartDate, setSymptomsStartDate] = React.useState<Date>();
-    const [hasSymptoms, setHasSymptoms] = React.useState<boolean>(false);
-    const [endInvestigationDate, setEndInvestigationDate] = React.useState<Date>();
+
     const {
         currentTab,
         setCurrentTab,
         confirmFinishInvestigation
-    } = useInvestigationForm();
+    } = useContent();
 
-    const startInvestigationDateVariables: StartInvestigationDateVariables = React.useMemo(() => ({ 
-            exposureDate, 
-            symptomsStartDate, 
-            hasSymptoms,
-            endInvestigationDate,
-            setExposureDate,
-            setSymptomsStartDate,
-            setHasSymptoms,
-            setEndInvestigationDate,
-        }),
-        [exposureDate, symptomsStartDate, hasSymptoms, endInvestigationDate,
-        setSymptomsStartDate, setExposureDate, setHasSymptoms, setEndInvestigationDate]
-    )
-    
     return (
         <div className={classes.content}>
-            <StartInvestigationDateVariablesProvider value={startInvestigationDateVariables}>
-                <InvestigationInfoBar/>
-                <div className={classes.interactiveForm}>
-                    <TabManagement
-                        currentTab={currentTab}
-                        setCurrentTab={setCurrentTab}
-                    />
-                    <div className={classes.buttonSection}>
-                        <Button 
-                            variant='contained' 
-                            color='primary' 
-                            className={classes.finishInvestigationButton} 
-                                onClick={() => {
-                                currentTab.id === LAST_TAB_ID ? confirmFinishInvestigation() :
-                                    setCurrentTab(tabs[currentTab.id + 1])
-                            }}>
-                                {currentTab.id === LAST_TAB_ID ? END_INVESTIGATION : CONTINUE_TO_NEXT_TAB}
-                        </Button>
-                    </div>
+            <InvestigationInfoBar/>
+            <div className={classes.interactiveForm}>
+                <TabManagement
+                    currentTab={currentTab}
+                    setCurrentTab={setCurrentTab}
+                />
+                <div className={classes.buttonSection}>
+                    <Button variant='contained' className={classes.finishInvestigationButton} onClick={() => {
```

Comment:
The  color='primary' prop needs to be here - I think you can just replace it here